conf/app.routes

```diff
@@ -269,6 +269,11 @@ POST        /lead-partner-entity
 ## PartyTypeResolverController
 GET         /resolve-party-type                                                                controllers.TradingNameResolverController.resolve
 
+
+## Dispatch From Warehouse Page
+GET         /using-fulfilment-warehouse                                                        controllers.registration.returns.DispatchFromWarehouseController.show
+POST        /using-fulfilment-warehouse                                                        controllers.registration.returns.DispatchFromWarehouseController.submit
+
```

Comment:
probably got duplicated during a rebase, not needed

Reply:
Good catch! updated